### PR TITLE
TMA-764: use POST for SSO

### PR DESCRIPTION
**Pull request by kubamahnert**

### CODE REVIEW

File: lib/gooddata/connection.rb [hunk 1]
```diff
@@ -112,15 +117,14 @@ def sso_url(login, provider, opts = DEFAULT_SSO_OPTIONS)
...
-      }
-
-      RestClient.get url, params do |response, _request, _result|
-        Rest::Client.connect_sso(:sst_token => URI.decode(response.cookies['GDCAuthSST']))
+    def connect_sso(login, provider, url, opts)
+      url, params = sso_url(login, provider, url)
+
+      RestClient::Request.execute(opts.merge({method: :post, url: url, payload: params})) do |response, _request, _result|
+        return Rest::Client.connect_sso(opts.merge({headers: {
+          x_gdc_authsst: response.cookies['GDCAuthSST'],
```
Comment (yenkins) [inline lib/gooddata/connection.rb]: Calls 'response.cookies' 2 times (DuplicateMethodCall)

Comment (yenkins) [inline lib/gooddata/connection.rb]: Refers to 'response' more than self (maybe move it to another class?) (FeatureEnvy)

Comment (yenkins) [inline lib/gooddata/connection.rb]: Style/IndentHash: Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.

File: lib/gooddata/connection.rb [hunk 2]
```diff
@@ -83,21 +83,26 @@ def sso_url(login, provider, opts = DEFAULT_SSO_OPTIONS)
       file_json.rewind
       file_signed = Tempfile.new('gooddata-sso-signed')
 
-      cmd = "gpg --no-tty --armor --yes -u #{login} --output #{file_signed.path} --sign #{file_json.path}"
+      cmd = "gpg --yes --no-tty --armor -u #{login} --output #{file_signed.path} --sign #{file_json.path}"
```
Comment (yenkins) [inline lib/gooddata/connection.rb]: Calls 'file_signed.path' 2 times (DuplicateMethodCall)

File: lib/gooddata/connection.rb [hunk 3]
```diff
@@ -112,15 +117,14 @@ def sso_url(login, provider, opts = DEFAULT_SSO_OPTIONS)
...
-      params = {
-        :x_gdc_request => "#{GoodData::Rest::Connection.generate_string}:#{GoodData::Rest::Connection.generate_string}"
-      }
-
-      RestClient.get url, params do |response, _request, _result|
-        Rest::Client.connect_sso(:sst_token => URI.decode(response.cookies['GDCAuthSST']))
+    def connect_sso(login, provider, url, opts)
+      url, params = sso_url(login, provider, url)
+
+      RestClient::Request.execute(opts.merge({method: :post, url: url, payload: params})) do |response, _request, _result|
```
Comment (yenkins) [inline lib/gooddata/connection.rb]: Refers to 'opts' more than self (maybe move it to another class?) (FeatureEnvy)

Comment (yenkins) [inline lib/gooddata/connection.rb]: Style/BracesAroundHashParameters: Redundant curly braces around a hash parameter.

Comment (yenkins) [inline lib/gooddata/connection.rb]: Style/SpaceInsideHashLiteralBraces: Space inside { missing.

Comment (yenkins) [inline lib/gooddata/connection.rb]: Style/SpaceInsideHashLiteralBraces: Space inside } missing.

File: lib/gooddata/connection.rb [hunk 4]
```diff
@@ -112,15 +117,14 @@ def sso_url(login, provider, opts = DEFAULT_SSO_OPTIONS)
...
-    def connect_sso(login, provider)
-      url = sso_url(login, provider)
-
-      params = {
-        :x_gdc_request => "#{GoodData::Rest::Connection.generate_string}:#{GoodData::Rest::Connection.generate_string}"
-      }
-
-      RestClient.get url, params do |response, _request, _result|
-        Rest::Client.connect_sso(:sst_token => URI.decode(response.cookies['GDCAuthSST']))
+    def connect_sso(login, provider, url, opts)
```
Comment (yenkins) [inline lib/gooddata/connection.rb]: Has 4 parameters (LongParameterList)

File: lib/gooddata/connection.rb [hunk 5]
```diff
@@ -66,7 +66,7 @@ def with_connection(options = nil, second_options = nil, third_options = {}, &bl
     # @param [Hash] opts Additional options
     # @option opts [Fixnum] :validity Validity in seconds from 'now'
     # @return [String] URL which can be used for SSO logging in
-    def sso_url(login, provider, opts = DEFAULT_SSO_OPTIONS)
+    def sso_url(login, provider, url = GoodData::Rest::Connection::DEFAULT_URL, opts = DEFAULT_SSO_OPTIONS)
```
Comment (yenkins) [inline lib/gooddata/connection.rb]: Has 4 parameters (LongParameterList)

Comment (yenkins) [inline lib/gooddata/connection.rb]: Has approx 20 statements (TooManyStatements)

File: lib/gooddata/rest/client.rb [hunk 6]
```diff
@@ -111,8 +111,7 @@ def connect(username, password = 'aaaa', opts = {})
         end
 
         def connect_sso(sso)
-          client = Client.new(sso)
-          client
+          @@instance = Client.new(sso)
```
Comment (yenkins) [inline lib/gooddata/rest/client.rb]: Declares the class variable '@@instance' (ClassVariable)

Comment (yenkins) [inline lib/gooddata/rest/client.rb]: Style/ClassVars: Replace class var @@instance with a class instance var.

File: lib/gooddata/rest/connection.rb [hunk 7]
```diff
@@ -210,13 +168,23 @@ def connect(username, password, options = {})
...
           get('/gdc/account/token', @request_params)
 
           @user = get(get('/gdc/app/account/bootstrap')['bootstrapResource']['accountSetting']['links']['self'])
           GoodData.logger.info("Connected using SST to server #{@server.url} to profile \"#{@user['accountSetting']['login']}\"")
           @auth = {}
           refresh_token :dont_reauth => true
+        elsif  options[:headers][:x_gdc_authsst]
+          @request_params = options[:headers]
+          @user = get('/gdc/app/account/bootstrap')['bootstrapResource']
+          GoodData.logger.info("Connected using SST to server #{@server.url} to profile \"#{@user['accountSetting']['login']}\"")
```
Comment (yenkins) [inline lib/gooddata/rest/connection.rb]: Calls '@server.url' 3 times (DuplicateMethodCall)

Comment (yenkins) [inline lib/gooddata/rest/connection.rb]: Calls '@user['accountsetting']' 2 times (DuplicateMethodCall)

Comment (yenkins) [inline lib/gooddata/rest/connection.rb]: Calls '@user['accountsetting']['login']' 2 times (DuplicateMethodCall)

Comment (yenkins) [inline lib/gooddata/rest/connection.rb]: Calls 'gooddata.logger' 8 times (DuplicateMethodCall)

Comment (yenkins) [inline lib/gooddata/rest/connection.rb]: Calls 'gooddata.logger.info("connected using sst to server #{@server.url} to profile \"#{@user['accountsetting']['login']}\"")' 2 times (DuplicateMethodCall)

File: lib/gooddata/rest/connection.rb [hunk 8]
```diff
@@ -210,13 +168,23 @@ def connect(username, password, options = {})
...
+          merge_headers!(headers)
           get('/gdc/account/token', @request_params)
 
           @user = get(get('/gdc/app/account/bootstrap')['bootstrapResource']['accountSetting']['links']['self'])
           GoodData.logger.info("Connected using SST to server #{@server.url} to profile \"#{@user['accountSetting']['login']}\"")
           @auth = {}
           refresh_token :dont_reauth => true
+        elsif  options[:headers][:x_gdc_authsst]
+          @request_params = options[:headers]
+          @user = get('/gdc/app/account/bootstrap')['bootstrapResource']
```
Comment (yenkins) [inline lib/gooddata/rest/connection.rb]: Calls 'get('/gdc/app/account/bootstrap')' 2 times (DuplicateMethodCall)

Comment (yenkins) [inline lib/gooddata/rest/connection.rb]: Calls 'get('/gdc/app/account/bootstrap')['bootstrapresource']' 2 times (DuplicateMethodCall)

File: lib/gooddata/rest/connection.rb [hunk 9]
```diff
@@ -210,13 +168,23 @@ def connect(username, password, options = {})
...
+            :x_gdc_authtt => options[:tt_token]
+          }
+          merge_headers!(headers)
           get('/gdc/account/token', @request_params)
 
           @user = get(get('/gdc/app/account/bootstrap')['bootstrapResource']['accountSetting']['links']['self'])
           GoodData.logger.info("Connected using SST to server #{@server.url} to profile \"#{@user['accountSetting']['login']}\"")
           @auth = {}
           refresh_token :dont_reauth => true
+        elsif  options[:headers][:x_gdc_authsst]
```
Comment (yenkins) [inline lib/gooddata/rest/connection.rb]: Calls 'options[:headers]' 3 times (DuplicateMethodCall)

File: lib/gooddata/rest/connection.rb [hunk 10]
```diff
@@ -210,13 +168,23 @@ def connect(username, password, options = {})
 
         # Reset old cookies first
         if options[:sst_token]
-          merge_headers!(:x_gdc_authsst => options[:sst_token])
+          headers = {
+            :x_gdc_authsst => options[:sst_token],
```
Comment (yenkins) [inline lib/gooddata/rest/connection.rb]: Calls 'options[:sst_token]' 2 times (DuplicateMethodCall)

File: lib/gooddata/rest/connection.rb [hunk 11]
```diff
@@ -210,13 +168,23 @@ def connect(username, password, options = {})
...
           @user = get(get('/gdc/app/account/bootstrap')['bootstrapResource']['accountSetting']['links']['self'])
           GoodData.logger.info("Connected using SST to server #{@server.url} to profile \"#{@user['accountSetting']['login']}\"")
           @auth = {}
           refresh_token :dont_reauth => true
+        elsif  options[:headers][:x_gdc_authsst]
+          @request_params = options[:headers]
+          @user = get('/gdc/app/account/bootstrap')['bootstrapResource']
+          GoodData.logger.info("Connected using SST to server #{@server.url} to profile \"#{@user['accountSetting']['login']}\"")
+          @auth = {}
+          refresh_token :dont_reauth => true
```
Comment (yenkins) [inline lib/gooddata/rest/connection.rb]: Calls 'refresh_token :dont_reauth => true' 3 times (DuplicateMethodCall)

File: lib/gooddata/connection.rb [hunk 12]
```diff
@@ -83,21 +83,26 @@ def sso_url(login, provider, opts = DEFAULT_SSO_OPTIONS)
...
       file_final.rewind
       final = file_final.read
 
-      "#{GoodData::Helpers::AuthHelper.read_server}/gdc/account/customerlogin?sessionId=#{CGI.escape(final)}&serverURL=#{CGI.escape(provider)}&targetURL=#{CGI.escape(opts[:url])}"
+      params = {
+        targetUrl: opts[:url],
+        ssoProvider: provider,
+        encryptedClaims: final
+      }
+      return url + '/gdc/account/customerlogin', params
```
Comment (yenkins) [inline lib/gooddata/connection.rb]: Style/RedundantReturn: Redundant `return` detected.

File: lib/gooddata/connection.rb [hunk 13]
```diff
@@ -112,15 +117,14 @@ def sso_url(login, provider, opts = DEFAULT_SSO_OPTIONS)
...
-        :x_gdc_request => "#{GoodData::Rest::Connection.generate_string}:#{GoodData::Rest::Connection.generate_string}"
-      }
-
-      RestClient.get url, params do |response, _request, _result|
-        Rest::Client.connect_sso(:sst_token => URI.decode(response.cookies['GDCAuthSST']))
+    def connect_sso(login, provider, url, opts)
+      url, params = sso_url(login, provider, url)
+
+      RestClient::Request.execute(opts.merge({method: :post, url: url, payload: params})) do |response, _request, _result|
+        return Rest::Client.connect_sso(opts.merge({headers: {
```
Comment (yenkins) [inline lib/gooddata/connection.rb]: Style/BracesAroundHashParameters: Redundant curly braces around a hash parameter.

Comment (yenkins) [inline lib/gooddata/connection.rb]: Style/SpaceInsideHashLiteralBraces: Space inside { missing.

File: lib/gooddata/connection.rb [hunk 14]
```diff
@@ -112,15 +117,14 @@ def sso_url(login, provider, opts = DEFAULT_SSO_OPTIONS)
...
+          x_gdc_authsst: response.cookies['GDCAuthSST'],
+          x_gdc_authtt: response.cookies['GDCAuthTT']
+        }}))
```
Comment (yenkins) [inline lib/gooddata/connection.rb]: Style/IndentHash: Indent the right brace the same as the first position after the preceding left parenthesis.

Comment (yenkins) [inline lib/gooddata/connection.rb]: Style/SpaceInsideHashLiteralBraces: Space inside } missing.

File: spec/integration/connection_spec.rb [hunk 15]
```diff
@@ -0,0 +1,29 @@
...
+    key = ENV['GD_SPEC_PASSWORD'] || ENV['BIA_ENCRYPTION_KEY']
+    File.write('dev-gooddata-sso.pub', GoodData::Helpers.decrypt(File.read('dev-gooddata-sso.pub.encrypted'), key))
+    File.write('rubydev_public.gpg', GoodData::Helpers.decrypt(File.read('rubydev_public.gpg.encrypted'), key))
+    File.write('rubydev_secret_keys.gpg', GoodData::Helpers.decrypt(File.read('rubydev_secret_keys.gpg.encrypted'), key))
+
+    `gpg --import dev-gooddata-sso.pub rubydev_public.gpg rubydev_secret_keys.gpg`
+  end
+
+  it 'can use SSO' do
+    rest_client = GoodData.connect_sso(login, 'test-ruby', GoodData::Environment::ConnectionHelper::DEFAULT_SERVER, {
```
Comment (yenkins) [inline spec/integration/connection_spec.rb]: Style/BracesAroundHashParameters: Redundant curly braces around a hash parameter.

File: spec/integration/connection_spec.rb [hunk 16]
```diff
@@ -0,0 +1,29 @@
...
+      server: GoodData::Environment::ConnectionHelper::DEFAULT_SERVER,
```
Comment (yenkins) [inline spec/integration/connection_spec.rb]: Style/IndentHash: Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.

File: spec/integration/connection_spec.rb [hunk 17]
```diff
@@ -0,0 +1,29 @@
...
+      verify_ssl: false
+    })
```
Comment (yenkins) [inline spec/integration/connection_spec.rb]: Style/IndentHash: Indent the right brace the same as the first position after the preceding left parenthesis.

File: spec/integration/connection_spec.rb [hunk 18]
```diff
@@ -0,0 +1,29 @@
...
+    user = rest_client.domain('staging2-lcm-prod').users(login, client: rest_client)
+    expect(user).to be_truthy
+  end
+end
+
```
Comment (yenkins) [inline spec/integration/connection_spec.rb]: Style/TrailingBlankLines: 1 trailing blank lines detected.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/845/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/712/ : FAILURE in 2m 29s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/979/ : SUCCESS in 1m 52s

File: lib/gooddata/connection.rb (hunk 5 above)
Comment (yenkins) [inline lib/gooddata/connection.rb]: Has approx 22 statements (TooManyStatements)

File: spec/integration/connection_spec.rb [hunk 19]
```diff
@@ -0,0 +1,35 @@
...
+    cipher = ENV['GD_SPEC_PASSWORD'] || ENV['BIA_ENCRYPTION_KEY']
+    KEYS.each do |key|
+      File.write(key, GoodData::Helpers.decrypt(File.read(key + '.encrypted'), cipher))
+    end
+
+    `gpg --import #{KEYS.join(' ')}`
+  end
+
+  it 'can use SSO' do
+    rest_client = GoodData.connect_sso(login, 'test-ruby', GoodData::Environment::ConnectionHelper::DEFAULT_SERVER, {
```
Comment (yenkins) [inline spec/integration/connection_spec.rb]: Style/BracesAroundHashParameters: Redundant curly braces around a hash parameter.

File: spec/integration/connection_spec.rb [hunk 20]
```diff
@@ -0,0 +1,35 @@
...
+    })
+    user = rest_client.domain('staging2-lcm-prod').users(login, client: rest_client)
+    expect(user).to be_truthy
+  end
+
+  after do
+    KEYS.each { |key| File.delete(key) }
+  end
+end
+
```
Comment (yenkins) [inline spec/integration/connection_spec.rb]: Style/TrailingBlankLines: 1 trailing blank lines detected.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/846/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/713/ : FAILURE in 37s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/980/ : SUCCESS in 1m 58s

File: lib/gooddata/connection.rb [hunk 21]
```diff
@@ -112,15 +118,18 @@ def sso_url(login, provider, opts = DEFAULT_SSO_OPTIONS)
...
-      RestClient.get url, params do |response, _request, _result|
-        Rest::Client.connect_sso(:sst_token => URI.decode(response.cookies['GDCAuthSST']))
+    def connect_sso(login, provider, url, opts)
+      url, params = sso_url(login, provider, url)
+
+      RestClient::Request.execute(opts.merge(method: :post, url: url, payload: params)) do |response, _request, _result|
+        return Rest::Client.connect_sso(
+          opts.merge(headers:
+                       {
+                         x_gdc_authsst: response.cookies['GDCAuthSST'],
```
Comment (yenkins) [inline lib/gooddata/connection.rb]: Calls 'response.cookies' 2 times (DuplicateMethodCall)

Comment (yenkins) [inline lib/gooddata/connection.rb]: Refers to 'response' more than self (maybe move it to another class?) (FeatureEnvy)

File: lib/gooddata/connection.rb [hunk 22]
```diff
@@ -112,15 +118,18 @@ def sso_url(login, provider, opts = DEFAULT_SSO_OPTIONS)
...
+                         x_gdc_authtt: response.cookies['GDCAuthTT']
+                       }
+          )
```
Comment (yenkins) [inline lib/gooddata/connection.rb]: Style/ClosingParenthesisIndentation: Align `)` with `(`.

Comment (yenkins) [inline lib/gooddata/connection.rb]: Style/MultilineMethodCallBraceLayout: Closing method call brace must be on the same line as the last argument when opening brace is on the same line as the first argument.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/848/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/715/ : FAILURE in 38s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/982/ : SUCCESS in 33s

File: lib/gooddata/connection.rb [hunk 23]
```diff
@@ -112,15 +118,19 @@ def sso_url(login, provider, opts = DEFAULT_SSO_OPTIONS)
...
-        Rest::Client.connect_sso(:sst_token => URI.decode(response.cookies['GDCAuthSST']))
+    def connect_sso(login, provider, url, opts)
+      url, params = sso_url(login, provider, url)
+
+      RestClient::Request.execute(opts.merge(method: :post, url: url, payload: params)) do |response, _request, _result|
+        return Rest::Client.connect_sso(
+          opts.merge(
+            headers:
+               {
+                 x_gdc_authsst: response.cookies['GDCAuthSST'],
```
Comment (yenkins) [inline lib/gooddata/connection.rb]: Calls 'response.cookies' 2 times (DuplicateMethodCall)

Comment (yenkins) [inline lib/gooddata/connection.rb]: Refers to 'response' more than self (maybe move it to another class?) (FeatureEnvy)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/849/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/716/ : SUCCESS in 37s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/983/ : SUCCESS in 32s

**Comment by kubamahnert** [comment]
extended test

File: spec/integration/connection_spec.rb [hunk 24]
```diff
@@ -0,0 +1,37 @@
...
+  it 'can use SSO' do
+    rest_client = GoodData.connect_sso(
+      login,
+      'test-ruby',
+      GoodData::Environment::ConnectionHelper::DEFAULT_SERVER,
+      server: GoodData::Environment::ConnectionHelper::DEFAULT_SERVER,
+      verify_ssl: false
+    )
+    user = rest_client.domain('staging2-lcm-prod').users(login, client: rest_client)
```
[the remark this answers is not in the document]
Comment (kubamahnert) [reply spec/integration/connection_spec.rb]: fixd

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/851/ : SUCCESS in 13s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/717/ : SUCCESS in 36s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/984/ : SUCCESS in 32s

**Comment by kubamahnert** [comment]
extended test

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/854/ : SUCCESS in 17s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1270/ : FAILURE in 55m 00s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/1034/ : UNSTABLE in 42m 52s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/858/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/723/ : SUCCESS in 40s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/990/ : SUCCESS in 33s

**Comment by kubamahnert** [comment]
extended test

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/859/ : SUCCESS in 29s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1271/ : ABORTED in 2h 00m 01s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/1035/ : UNSTABLE in 43m 04s

**Comment by kubamahnert** [comment]
extended test

**Comment by kubamahnert** [comment]
extended test - development

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/861/ : SUCCESS in 16s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1273/ : ABORTED in 5m 50s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/1037/ : UNSTABLE in 40m 59s

**Comment by kubamahnert** [comment]
extended test

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/879/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1278/ : ABORTED in 1h 18m 09s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/1042/ : UNSTABLE in 45m 44s

**Comment by kubamahnert** [comment]
extended test

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/899/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1285/ : FAILURE in 52m 18s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/1049/ : UNSTABLE in 39m 23s

**Comment by kubamahnert** [comment]
extended test

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/900/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1286/ : FAILURE in 55m 34s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/1050/ : UNSTABLE in 43m 22s

**Comment by Hahihula** [comment]
extended test

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/902/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1288/ : SUCCESS in 56m 42s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/1052/ : UNSTABLE in 42m 36s

**Comment by Hahihula** [comment]
extended test

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/904/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1291/ : SUCCESS in 55m 30s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/1055/ : SUCCESS in 42m 54s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/319/ : SUCCESS in 28s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/755/ : SUCCESS in 2m 23s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1023/ : SUCCESS in 36s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1292/ : SUCCESS in 54m 36s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/1056/ : SUCCESS in 42m 23s

**Pull request merged**